cmd/kubeconfig/create_test.go

```diff
@@ -44,6 +44,7 @@ func TestKubeconfigCreate(t *testing.T) {
 		"--config", "-",
 		"--data-dir", k0sVars.DataDir,
 		"kubeconfig", "create", "test-user",
+		"--context-name", "my-cluster",
```

Comment:
Can we add an assertion that this is the current context in the generated kubeconfig?